prog/vnet/cert_nexus.rb

```diff
@@ -131,4 +141,20 @@ def dns_record_name
   def dns_zone
     @dns_zone ||= DnsZone[cert.dns_zone_id]
   end
+
+  def create_self_signed_cert(duration)
```

Comment:
we have similar functionality in our utils, which is used by Postgres and Minio, but that doesn't produce certs compatible for this use-case.